Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -60,6 +60,9 @@ public final class ViewAnnotationManager {
     /// The default value is true, and setting this value to false will disable the validation.
     public var validatesViews = true
 
+    /// The complete list of annotations associated with the receiver.
+    public var annotations: [UIView] { Array(viewsById.values) }
```

Comment:
Should we also return `ViewAnnotationOptions` associated with the view? Otherwise, engine (developer) would need to iterate twice:

```
for (view : manager.annotations) {
  coordinate = manager.options(view).geometry // lookup 
}
```

Reply:
that makes sense, I have amended the api to return list of added annotation view and its options, thank you!